app/assets/javascripts/discourse/app/components/edit-category-topic-template.js

```diff
@@ -1,15 +1,51 @@
 import { buildCategoryPanel } from "discourse/components/edit-category-panel";
-import { observes } from "discourse-common/utils/decorators";
+import discourseComputed, { observes } from "discourse-common/utils/decorators";
 import { schedule } from "@ember/runloop";
+import FormTemplate from "admin/models/form-template";
+import { action } from "@ember/object";
+import { notEmpty } from "@ember/object/computed";
 
 export default buildCategoryPanel("topic-template", {
   // Modals are defined using the singleton pattern.
   // Opening the insert link modal will destroy the edit category modal.
   showInsertLinkButton: false,
+  showFormTemplate: notEmpty("category.form_template_ids"),
```

Comment:
Getting `computed property was just overridden` deprecation warning when toggling this property with `<DToggleSwitch/>`. Is there a way we can get this to have a default initial value, but toggleable without the deprecation?